src/assets/i18n/fr_FR.json

```diff
@@ -891,6 +891,9 @@
   "product.ready_for_shipment.within24": "Habituellement prêt à être expédié dans les 24 heures",
   "product.remove_all_product_filters.text": "Réinitialiser tous les filtres",
   "product.retailSet.products.heading": "Ce jeu de produits contient",
+  "product.reviews.heading": "Notations",
```

Comment:
_Reviews_ and _submit_ are translated differently according to our translation memory system. Please ask Nicolas for review of fr_FR.json